native/kotlin/api/kotlin/src/integrationTest/kotlin/WordPressApiCacheTest.kt

```diff
@@ -0,0 +1,16 @@
+package rs.wordpress.api.cache.kotlin
+
+import org.junit.jupiter.api.Test
```

Comment:
```suggestion
import kotlinx.coroutines.test.runTest
import org.junit.jupiter.api.Test
```

This will be necessary when `performMigrations` is updated to be a suspended function.

Reply:
[9acf4ef](https://github.com/Automattic/wordpress-rs/pull/903/commits/9acf4efa620475ebe9757b3ffb835e45c23821d7)